tests/security/yast2_apparmor/manually_add_profile.pm

```diff
@@ -51,6 +51,7 @@ sub run {
     # "marked as a program that should not have its own profile",
     # it should be failed
     assert_and_click("AppArmor-Manually-Add-Profile");
```

Comment:
Send_key then do you still need click?

Reply:
> Send_key then do you still need click?

Yes, need the click to select the item then launch.